src/core/resources/transactions/transaction.ts

```diff
@@ -146,7 +172,7 @@ const getCustomChainTransaction = async ({
         from: transaction.from as Address,
         to: transaction.to as Address,
         data: transaction.data,
-        value: transaction.value.toString(),
+        value,
```

Comment:
i don't think we want to do this, i assume that value is in wei

is not breaking other places of the extension?

Reply:
`parseTransaction` formats the value like that, is the customChainTransaction used differently then a normal supported transaction?
didn't notice it breaking anything but possible

Reply:
right

<img width="646" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/12115171/3c43bcc4-4bcc-4c29-a689-7f4e6dd554fe">

i don't get why is working then 🤔

Reply:
`providerTx` is early returning before getting to `parseTransaction`